ScattReport.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Scatt.java
+ *  
+ * @author Kara Beason
+ * @author Cydney Caldwell
+ * @author Michelle Melton
+ * @version Mar 2017
+ */
+
+/**
+ * Scratch Code Analysis Tool for Teachers (SCATT). 
+ * 
+ * @author Kara Beason
+ * @author Cydney Caldwell
+ * @author Michelle Melton
+ * @version Mar 2017
+ */
+public class ScattReport
+{
+    /**
+     * Main method for ScattReport class.
+     *
+     * @param args 
+     */
+    public static void main(String[] args)
```

Comment:
The Report class does not need a main method - that will be contained in Scatt.java.